controllers/topic-controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectTopics } = require("../models/topic-model.js");
+
+const getTopics = (request, response) => {
+  selectTopics().then(({ rows: topics }) => {
+    //console.log(topics);
```

Comment:
Remove this console.log before proceeding, please.

Reply:
Looks like it is still here - please remove